build/nse/ucnf-kiknos/vppagent/Dockerfile

```diff
@@ -2,7 +2,6 @@ ARG VPP_AGENT
 
 FROM golang:alpine as build
 RUN apk --no-cache add git
-ENV PACKAGEPATH=github.com/networkservicemesh/networkservicemesh/
```

Comment:
I am not sure we need to delete this. I don't exactly know why we ned this env but it is defined in every NSM docker image, so maybe there is a reason to have this. Also this change has nothing to do with the PR intent

Reply:
Done